### ISSUE TITLE
downgrading AbstractPlotting v0.15.6 ⇒ v0.13.11

### ISSUE THREAD

**Issue by lazarusA** [JuliaPlots/GeoMakie.jl#64]
Is it possible to simply increase the upper bound for AbstractPlotting without breaking the functionality of the package?, or is not that straightforward ?

I'm looking forward to make a gallery :D, if possible. 
Cheers,

**Comment by SimonDanisch** [comment]
It can't be super hard to upgrade GeoMakie to the newest AbstractPlotting, since we haven't changed much of the internals.
I don't have time to maintain GeoMakie right now, and @asinghvi17 doesn't seem to either.
So I guess, we need to leave upgrading to some one else at the moment. The good news is, that it should be pretty straight forward for anyone familiar with Makie!

**Comment by visr** [comment]
I don't have much time either, but I'd like GeoMakie to keep working if possible, so I could help out with merging simple PRs and such, though I don't have the rights for that now.

**Comment by SimonDanisch** [comment]
Thanks @visr!
I also don't mind merging PRs, and that's not really the bottleneck since there are currently no PRs to fix tests and make sure stuff works...
If I remember correctly @datseris said GeoMakie is broken right now, so I also haven't merge the CompatHelper PRs. 
Maybe I should simply do that to lower the burden for anyone trying to get things running...

**Comment by Datseris** [comment]
Hi yes, GeoMakie is indeed broken I have tested it internally. For AbPlot 0.14 the transition was indeed simple, but now, with AbPlot 0.15, it will take a bit of effort and of course someone that knows Makie a bit to update GeoMakie. It is also the fact that Proj4.jl is also changing, and the discussion that Anshul had initialized some time ago to moving to something more fundamental like GeometryBasics...? I'm not sure.

I can say for sure I also do not have the bandwidth to contribute here, even though I too use GeoMakie in a project, and live with having AbPlot in versions before the merge with MakieLayout. Would be super useful for me to have GeoMakie updated, given that the rest of my activities e.g. InteractiveChaos have been updated to AbPlot 0.15.
